integration/fuse/src/main/java/alluxio/jnifuse/AbstractFuseFileSystem.java

```diff
@@ -132,27 +132,49 @@ public void umount() {
   }
 
   public int openCallback(String path, ByteBuffer buf) {
-    FuseFileInfo fi = new FuseFileInfo(buf);
-    return open(path, fi);
+    try {
+      return open(path, FuseFileInfo.wrap(buf));
+    } catch (Exception e) {
+      e.printStackTrace();
```

Comment:
`e.printStackTrace();` prints to stdout, not appropriate for production code.
Use `LOG.error("Failed to open {}", path, e)` here so the stack trace will still be printed.
with claiming  `private static final Logger LOG = LoggerFactory.getLogger(AbstractFuseFileSystem.class);` in the beginning.